flexibleengine/resource_flexibleengine_cce_node_v3.go

```diff
@@ -61,6 +62,16 @@ func resourceCCENodeV3() *schema.Resource {
 				Optional: true,
 				ForceNew: true,
 			},
+			"user_tags": {
```

Comment:
better to just name it to tags, and should be conflict with lables: ConflictsWith: []string{"labels"} and deprecated it.

Reply:
fixed